README.md

```diff
@@ -1,2 +1,14 @@
 # image-analysis-python
 Materials for the SWC course on Image Analysis in Python
+
+## Installation
+
+Firstly install [miniconda](https://docs.brainglobe.info/cellfinder/using-conda) (or anaconda) from [here](https://docs.conda.io/en/latest/miniconda.html). 
+
+Then run the following commands in a terminal:
+```bash
+conda env create --name image-analysis python=3.10 nb_conda_kernels
```

Comment:
Napari stopped responding while I was running:
```python
detected_cells = cellfinder_run(signal_array,background_array,voxel_sizes)
```

Is this normal or is it due to the tensorflow-not-seeing-the-GPU problem?

### EDIT
I restarted the kernel and rerun the notebook and it ran to completion without crashing

Reply:
Thanks, I've made those changes.

> Maybe we can ask them to pre-download it? Otherwise, with everyone connected to the same network access point on the day of the course, it may not work smoothly.
On the other hand, most people will not bother with pre-downloading

I'll add a note about this when I message attendees in advance of the course. I assumed most people wouldn't follow along anyway, but they have the material if they want to look at it later on.